lib/panel-league/constant.js

```diff
@@ -1,6 +1,6 @@
 module.exports.scoring = {
   tetrisAttack: {
-    combo: [20, 30, 50, 60, 70, 80, 100, 140, 170],
+    combo: [20, 30, 50, 60, 70, 80, 100, 140, 170, 210, 250, 290, 340, 390, 440, 490, 550, 610, 680, 750, 820, 900, 980, 1060, 1150, 1240, 1330],
```

Comment:
This line is now very very long. Are you certain you do not want to read this data from external file?